src/main/java/com/manning/gia/todo/repository/mapdb/ToDoMapdbRepository.java

```diff
@@ -52,6 +53,11 @@ public void delete(ToDoItem toDoItem) {
         todos.remove(toDoItem.getId());
     }
 
+    @Override
+    public List<ToDoItem> findCompleted() {
+        return findAll().stream().filter(ToDoItem::isCompleted).collect(Collectors.toList());
+    }
```

Comment:
If we're going to use this method in perfomance tests, shouldn't we use DB engine to do the filtering?

Reply:
AFAIK it does not have query engine.
@ui-valts-mazurs, @ui-roberts-blums please let me know if there is.

Reply:
Yeah, didn't see any API for querying.